services/wallet/service.go

```diff
@@ -475,6 +475,13 @@ func (service *Service) LinkZebPayWallet(ctx context.Context, walletID uuid.UUID
 	}
 
 	if err := claims.validate(time.Now()); err != nil {
+		// KYC issue
+		if errors.Is(err, errZPInvalidKyc) {
+			return handlers.WrapError(
+				errors.New("user kyc did not pass"),
```

Comment:
I suggest we declare this error as a sentinel error (i.e. a global variable like other errors), for a few reasons:
- it will be easier to compare against during testing
- we don't do unnecessary work by recreating what could be reused.

Reply:
think i have a working solution in https://github.com/brave-intl/bat-go/pull/1958/commits/d17c392be55d3a181feec228972eaac69b9eee03